### PR TITLE
MH-12934, Update translations

### PR DESCRIPTION
**Pull request by lkiesow**
- Update the translations to the current state on Crowdin.
- Add new languages: Filipino, Tagalog, Turkish

### CODE REVIEW

File: modules/admin-ui/package-lock.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
       "version": "4.0.2",
-      "resolved": "https://nexus.opencast.org/nexus/content/repositories/npmjs/JSV/-/JSV-4.0.2.tgz",
+      "resolved": "http://nexus.opencast.org/nexus/content/repositories/npmjs/JSV/-/JSV-4.0.2.tgz",
       "integrity": "sha1-0Hf2glVx+CEy+d/67Vh7QCn+/1c=",
       "dev": true
     },
     "abbrev": {
       "version": "1.1.1",
-      "resolved": "https://nexus.opencast.org/nexus/content/repositories/npmjs/abbrev/-/abbrev-1.1.1.tgz",
+      "resolved": "http://nexus.opencast.org/nexus/content/repositories/npmjs/abbrev/-/abbrev-1.1.1.tgz",
```
Comment (staubesv) [inline modules/admin-ui/package-lock.json]: Hi @lkiesow This change seems not related to updating translations. Why is it included in this PR?

**Review by staubesv — CHANGES_REQUESTED** [review]
Hi @lkiesow 

I've found two issues:

- Theodul Player needs likely to be adapted to support the new languages (see https://bitbucket.org/opencast-community/opencast/pull-requests/1792/mh-12515-add-language-support-for/diff#Lmodules/matterhorn-engage-theodul-core/src/main/resources/ui/js/engage/core.jsT184, for example)
- Changes to package-lock.json seem not related to this PR

**Comment by lkiesow** [comment]
- Updated list of languages supported by Theodul (this includes sorting the list and removing all languages no longer supported)
- removed `package-lock.json`

**Review by staubesv — APPROVED** [review]

**Comment by staubesv** [comment]
Hi @lkiesow Please note that I've merged this without waiting for CI to complete (was running for more than 2 hours).
